pkg/security/probe/probe_linux.go

```diff
@@ -48,3 +60,26 @@ func (p *Probe) Origin() string {
 	}
 	return EBPFOrigin
 }
+
+func NewAgentContainerContext() (*events.AgentContainerContext, error) {
+	pid := os.Getpid()
+
+	procProcess, err := gopsutilProcess.NewProcess(int32(pid))
+	if err != nil {
+		return nil, err
+	}
+	createTime, err := procProcess.CreateTime()
+	if err != nil {
+		return nil, err
+	}
+	acc := &events.AgentContainerContext{
+		CreatedAt: uint64(createTime),
+	}
+
+	cid, err := utils.GetProcContainerID(uint32(pid), uint32(pid))
+	if err != nil { // could fail if not in container
+		return acc, nil
```

Comment:
Shouldn't this be returning `nil` and the error here?

Reply:
Tru. I though the func returns an error if no containerID is found, but it doesn't. Fixed on last commit. Thanks!